modules/microshift-low-latency-config-yaml.adoc

```diff
@@ -0,0 +1,71 @@
+<2> The name of the policy to use. Valid values are `none` and `static`. Requires the `CPUManager` feature gate to be enabled. Default value is `none`.
+<3> A set of `key=value` for setting extra options that fine tune the behavior of the `CPUManager` policies.  Default: null. Requires both the `CPUManager` and `CPUManagerPolicyOptions` feature gates to be enabled.
+<4> The name of the policy used by Memory Manager. Case-sensitive. Default value is `none`. Requires the `MemoryManager` feature gate to be enabled.
+<5> Required. The `reservedSystemCPUs` value must be the inverse of the offlined CPUs because both combined must account for all of the CPUs on the system. This parameter is essential to dividing the management and application workloads. Use this parameter to define a static CPU set for the host-level system and Kubernetes daemons, plus interrupts and timers. Then the rest of the CPUs on the system can be used exclusively for workloads.
+<6> The value in `reservedMemory[0].limits.memory`, `1100` Mi in this example, is equal to `kubeReserved.memory` + `systemReserved.memory` + `evictionHard.memory.available`.
+<7> The `evictionHard` parameters define under which conditions the kubelet evicts pods. When you change the
+default value of only one parameter for the `evictionHard` stanza, the default values of other parameters are not inherited and are set to zero. Provide all the threshold values even when you want to change just one.
+<8> The `imagefs` is a filesystem that container runtimes use to store container images and container writable layers. In this example, the `evictionHard.imagefs.available` parameter means that the pod is evicted when the available space of the image filesystem is less than 15%.
+<9> In this example, the `evictionHard.memory.available` parameter means that the pods are evicted when the available memory of the node drops below 100MiB.
+<10> In this example, the `evictionHard.nodefs.available` parameter means that the pods are evicted when the main filesystem of the node has less than 10% available space.
+<11> In this example, the `evictionHard.nodefs.inodesFree` parameter means that the pods are evicted when more than 15% of the node's main filesystem's inodes are in use.
+
+.Verification
+
+. After you complete the next steps and restart the host, you can use a root-access account to check that your settings are in the `config.yaml` file in the `/var/lib/microshift/resources/kubelet/config/` directory.
+
+.Next steps
+. Enable workload partitioning.
+. Tune your operating system. For example, configure and activate a TuneD profile.
+. Optional: Configure automatic enablement of your TuneD profile.
+. Optional: If you are using the x86_64 architecture, you can install {op-system-rt-kernel}.
+. Prepare your {microshift-short} workloads for low latency.
```

Comment:
Do we want to include links to make sure folks go to the right places?

Reply:
Would if I could!